assets/wix/files.wxs

```diff
@@ -3063,8 +3063,11 @@
       <Component Id="cmp2BB002B115C247FFB3353C80CAAC10B3">
         <File Id="filF1771BA6F8A140FFB77E7558D035A0F2" KeyPath="yes" Source="$(var.ProductSourcePath)\System.IO.Compression.Native.dll" />
       </Component>
-      <Component Id="cmpD9AA61B3F5F44A7B99E535A712E8310D">
-        <File Id="fil16547FB6A5EC4C6791FFFF5D561F2D31" KeyPath="yes" Source="$(var.ProductSourcePath)\mscordaccore_$(var.FileArchitecture)_$(var.FileArchitecture)_6.0.21.10212.dll" />
+      <Component Id="cmp3547A55EA9D04784B4CF2FC717447E16">
+        <File Id="fil463DDA419769417A8D5A34C433E194F2" KeyPath="yes" Source="$(var.ProductSourcePath)\System.Speech.dll" />
```

Comment:
Just curious, why this `System.Speech.dll` is mentioned here?

Reply:
Comes with the framework i guess

Reply:
This should be reported to .Net Runtime repo.